taui/src/selectors/neighborhoods-sorted-with-routes.js

```diff
@@ -23,9 +27,33 @@ export default createSelector(
       const active = route.active
       const segments = route.routeSegments
       const time = travelTimes[index]
-      return Object.assign({active, segments, time}, n)
-    }), n => n.segments && n.segments.length)
 
-    return sortBy(neighborhoodsWithRoutes, 'time')
+      // Map weighting values to percentages
+
+      // Routable neighborhoods outside the max travel time window will be filtered out.
+      // Smaller travel time is better; larger timeWeight is better (reverse range).
+      const timeWeight = time < MAX_TRAVEL_TIME ? scale(time, 0, MAX_TRAVEL_TIME, 1, 0) : 1
+
+      // Normalize rent differential to fair market value to a range;
+      // any value outside the range will be given the min/max value for the range.
+      const useRentDiff = n.properties.rent_diff > RENT_DIFF_RANGE ? RENT_DIFF_RANGE
```

Comment:
It'll be more verbose, but I think that a multi-line if-else would be preferable here; the nested ternary operators make this a bit tough to follow (at least, it took me longer to read this than I think it would have if it had been written out).